Submitted By: Ken Moffat <ken at linuxfromscratch dot org>
Date: 2006-12-29
Initial Package Version: 1.0.10
Upstream Status: unknown
Origin: From gentoo
Description: Allows nfs-utils to compile with the sanitised headers from linux-2.6.18 which omit UTS_RELEASE

--- nfs-utils-1.0.10/tools/getkversion/getkversion.c.orig	2006-12-29 23:34:16.000000000 +0000
+++ nfs-utils-1.0.10/tools/getkversion/getkversion.c	2006-12-29 23:34:38.000000000 +0000
@@ -14,6 +14,8 @@
 int
 main(void)	/* This is for Dan Popp ;) */
 {
+#ifdef UTS_RELEASE
 	printf("%s\n", UTS_RELEASE);
+#endif
 	return 0;
 }
